### PR TITLE
Refresh and restructure assets documentation

### PR DESCRIPTION
**Pull request by catenocrypt**

### CODE REVIEW

File: assets/add_new_asset.md [hunk 1]
```diff
@@ -96,92 +142,186 @@ Token repository [https://github.com/trustwallet/assets](https://github.com/trus
...
+- [Submit form for review](https://docs.google.com/forms/d/e/1FAIpQLSd5p9L78zKXIiu9E5yFRPf5UkvsLZ7TbUDLFBRIi1qMd8Td4A/viewform)
+
+### Staking validators requirements
+#### General requirements
+1. Add validator basic information to the bottom of the list, see example for: [Kava](https://github.com/trustwallet/assets/tree/master/blockchains/kava/validators/list.json), [Cosmos](https://github.com/trustwallet/assets/tree/master/blockchains/cosmos/validators/list.json), [Tezos](https://github.com/trustwallet/assets/tree/master/blockchains/tezos/validators/list.json), [Tron](https://github.com/trustwallet/assets/tree/master/blockchains/tron/validators/list.json), [Solana](https://github.com/trustwallet/assets/tree/master/blockchains/solana/validators/list.json), [Harmony](https://github.com/trustwallet/assets/tree/master/blockchains/harmony/validators/list.json)
+2. Add validator logo image to `blockchains/<chain>/validators/assets/<validator_address>/logo.png` [see images requirements](#image-requirements)
+3. Check chain [specific](#validators-specific-requirements) requirements 
+
+#### Validators specific requirements
+##### Tezos
```
Comment (vikmeup) [inline assets/add_new_asset.md:206]: we can remove tezos part, not needed

File: assets/add_new_asset.md [hunk 2]
```diff
@@ -14,77 +49,88 @@ Token repository [https://github.com/trustwallet/assets](https://github.com/trus
...
 │   │   └──assets
 │   │   │  └──0x0a2D9370cF74Da3FD3dF5d764e394Ca8205C50B6 // address folder
-│   │   │     └──logo.png // address logo
+│   │   │     └──logo.png  // token logo
+|   |   |     └──info.json // optional token info
 │   │   └──info
-│   │      └──logo.png // coin logo
+│   │   │  └──logo.png  // chain coin logo
+|   |   │  └──info.json // chain coin info
+│   │   └──whitelist.json // allow list for tokens
```
Comment (vikmeup) [inline assets/add_new_asset.md:107]: why we did not rename it yet to `allowlist` and `denylist`? this is one of the first things I have suggested to do?

File: assets/add_new_asset.md (hunk 1 above)
Comment (catenocrypt) [reply assets/add_new_asset.md]: done

File: assets/add_new_asset.md (hunk 2 above)
Comment (catenocrypt) [reply assets/add_new_asset.md]: I wanted to finish first changes with functional improvements, and this rename has to be also coordinated with backend.  It's done now, renamed here too.

**Pull request merged**